lib/smart_answer_flows/benefit-cap-calculator.rb

```diff
@@ -8,6 +8,18 @@ def define
 
       config = Calculators::BenefitCapCalculatorConfiguration.new
 
+      # Routing question
+      multiple_choice :choose_cap_to_calculate? do
+        option :default
+        option :future
+
+        save_input_as :chosen_cap
```

Comment:
Similar to my comment about the name of this question, I think this might be clearer if it was named `period_to_check`.